fe/fe-core/src/main/java/org/apache/doris/catalog/Tablet.java

```
@@ -314,12 +314,7 @@ public List<Replica> getQueryableReplicas(long visibleVersion, Map<Long, Set<Lon
                 continue;
             }
 
-            // Skip the missing version replica
-            if (replica.getLastFailedVersion() > 0 && !allowFailedVersion) {
-                continue;
-            }
-
-            if (!replica.checkVersionCatchUp(visibleVersion, false)) {
```

if be not found rowset [1,  visible version],  be maybe also throw a error